lib/bot.js

```diff
@@ -138,13 +138,17 @@ class Bot {
       const colorIndex = (username.charCodeAt(0) + username.length) % NICK_COLORS.length;
```

Comment:
Maybe put these two lines into the `if` too? No need to create `colorIndex` and `coloredUsername` if they aren't gonna be used.